torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -125,13 +125,13 @@ PyObject* tensor_to_numpy(const at::Tensor& tensor) {
   return array.release();
 }
 
-at::Tensor tensor_from_numpy(PyObject* obj) {
+at::Tensor tensor_from_numpy(PyObject* obj, bool copy_numpy/*=false*/) {
   if (!PyArray_Check(obj)) {
     throw TypeError("expected np.ndarray (got %s)", Py_TYPE(obj)->tp_name);
   }
   auto array = (PyArrayObject*)obj;
 
-  if (!PyArray_ISWRITEABLE(array)) {
+  if (!PyArray_ISWRITEABLE(array) && !copy_numpy) {
```

Comment:
Would you add a comment here explaining why this warning isn't need when copy_numpy is true?